ntoskrnl/ke/amd64/except.c

```diff
@@ -119,6 +119,9 @@ KiDispatchExceptionToUser(
     /* Get pointer to the usermode context, exception record and machine frame */
     UserStack = (PKUSER_EXCEPTION_STACK)UserRsp;
 
+    /* Enable interrupts */
+    _enable();
```

Comment:
In the `_SEH2_EXCEPT` / `_SEH2_YIELD(return);` case below, there is no need to re-disable the interrupts?